### PR TITLE
fix(components): [date-picker] date-table-cell namespace

### PR DESCRIPTION
**Pull request by YunYouJun**
- fix `date-picker` `date-table-cell` namespace
- fix `scrollbar` namespace

---

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by sxzz — APPROVED** [review]

File: docs/components.d.ts [hunk 1]
```diff
@@ -44,7 +44,6 @@ declare module '@vue/runtime-core' {
     Playground: typeof import('./.vitepress/vitepress/components/icons/playground.vue')['default']
     Resource: typeof import('./.vitepress/vitepress/components/globals/resource.vue')['default']
     ResourceSvg: typeof import('./.vitepress/vitepress/components/home/svg/resource-svg.vue')['default']
-    Right: typeof import('./.vitepress/vitepress/components/sponsors/right.vue')['default']
```
[the remark this answers is not in the document]
Comment (YunYouJun) [reply docs/components.d.ts:47]: This file doesn't exits. I think it should be removed.

[the remark this answers is not in the document]
Comment (sxzz) [reply docs/components.d.ts:47]: Ok, fair enough.

[the remark this answers is not in the document]
Comment (sxzz) [reply docs/components.d.ts:47]: And please try build docs again to ensure `docs/components.d.ts` is up to date.

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
@YunYouJun Please rebase dev for getting the updated actions behavior

**Pull request merged**